nilearn/plotting/displays.py

```diff
@@ -218,6 +236,89 @@ def draw_position(self, size, bg_color, **kwargs):
         # the cuts since we are taking the max along one axis
         pass
 
+    def _add_graph_nodes(self, nodes_coords, node_color, node_size, **kwargs):
+        """Plot graph nodes"""
+        if 's' in kwargs:
+            raise ValueError("Please use 'node_size' and not 'node_kwargs' "
+                             "to specify node sizes")
+        if 'c' in kwargs:
+            raise ValueError("Please use 'node_color' and not 'node_kwargs' "
+                             "to specify node colors")
+        if node_size is None:
+            node_size = 50
+        if node_color is None:
+            node_color = 'green'
+
+        nodes_coords_2d = _coords_3d_to_2d(nodes_coords, self.direction)
+
+        first_coords, second_coords = zip(*nodes_coords_2d)
+
+        defaults = {'marker': 'o',
+                    'zorder': 1000}
+        for k, v in defaults.items():
+            kwargs.setdefault(k, v)
+
+        self.ax.scatter(first_coords, second_coords, s=node_size, c=node_color,
```

Comment:
> Hmmm. It would be cool to have a sagittal view for the left hemisphere,
> another one for the right hemisphere, and then an axial view for the
> connexions between both of them.

Yes. I can see a design where there would be an Axes that projects to the
left hemisphere and one to the right. As in 'R', 'L'. We'll brain storm
about this later.

> Regarding depth, it would be nice to be able to change the size of the
> nodes depending on the depth so that further nodes are smaller. We
> can't do that yet because the depth depends on the view (or we should
> do manually the 3 slices).

+1 for having an option 'node_color="depth"'. In a later PR.

Do you want to add 2 enhancements tickets for these features? I think
that they are good ideas.